lmp/bb-config.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+
```

Comment:
I guess this shouldn't be here?

Reply:
ops

thanks for the review @ricardosalveti